lib/vsc/utils/fancylogger.py

```diff
@@ -431,7 +440,7 @@ def getRootLoggerName():
         return "not available in optimized mode"
 
 
-def logToScreen(enable=True, handler=None, name=None, stdout=False):
+def logToScreen(enable=True, handler=None, name=None, stdout=False, colorize='never'):
```

Comment:
well no, 
the point of vsc-base is that it doesn't have any external dependencies, 
things that have dependencies go into `vsc-utils`